src/arrays/ladderops.jl

```diff
@@ -22,10 +22,22 @@
     before_eye(lens, pivot) = speye(prod(lens[[1:pivot-1]]))
     after_eye(lens, pivot) = speye(prod(lens[[pivot+1:length(lens)]]))
     function eye_sandwich(lens, pivot, op)
-        return kron(before_eye(lens, pivot), 
-                    op, 
+        return kron(before_eye(lens, pivot),
+                    op,
                     after_eye(lens, pivot))
     end
 
+function positionop(n::Int)
+    cop = raiseop(n)
+    return (cop+cop')/sqrt(2.)
```

Comment:
We should use `scale!` here since `+` already makes a copy and `/` would make another one (same for `momentumop` below).